LPS-77699 Update Translations

sandrodw3 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

Commit:
LPD-1634 Track multiple saml_message_id

manuelecastro committed:
LPD-1634 Remove oldest entry when it reaches the limit

ChrisKian committed:
LPD-1634 Let's make this value a bit bigger, and configurable

ChrisKian committed:
LPD-1634 buildLang

ChrisKian committed:
LPD-1634 Just provide the initialCapacity arugument to the constructor, and let the HashMap deal with it

manuelecastro committed:
LPD-1634 Use LRUMap instead

manuelecastro committed:
LPD-1634 The SAML_SSO_REQUEST_CONTEXT is not being binded when it is IdP initiated, so when given a RelayState, it will not redirect to it.

ChrisKian committed:
LPD-1634 Store the IdP initiated SSO contexts individually, otherwise only the first one will be used.  We can do this by creating a unique ID to map the SSOContext, which will be stored and retrieved via a parameter.

manuelecastro committed:
LPD-1634 Add tests

manuelecastro committed:
LPD-1634 SemVer

# breaking

## What modules/dxp/apps/saml/saml-api/src/main/java/com/liferay/saml/runtime/configuration/SamlConfiguration.java

modules/dxp/apps/saml/saml-api/src/main/java/com/liferay/saml/runtime/configuration/SamlConfiguration.java now includes a new @Meta.AD saml-max-saml-sso-request-contexts.

## Why

Make maximum SAML SSO request contexts configurable.

----

ChrisKian committed:
LPD-1634 Let's reuse the saml_message_id param instead, for simplicity.  We will also need to make sure the InOutOperationContext is not null before continuing, since IdP initiated SSO doesn't set this context.

ChrisKian committed:
LPD-1634 Fix tests

ChrisKian committed:
LPD-1634 Reuse samlSsoRequestContext variable, because decoding twice will always result in a null value since we remove the entry from the map after decoding the first time.

ChrisKian committed:
LPD-1634 Some test changes are required since LPD-39115 has been merged

ChrisKian committed:
LPD-1634 ant update-portal-osgi-configuration-properties

gergelyszalay committed:
LPD-42538 add LPD-36446 FF

gergelyszalay committed:
LPD-42542 Add View Usages to the actions menu.

gergelyszalay committed:
LPD-42542 Create view_file_entry_usages and MVCRender

Commit:
LPD-22831 Create upgrade process to update userId for ObjectDefinition, ObjectField, ObjectRelationship, ListTypeDefinition and ListTypeEntry

Co-authored-by: Marcela Cunha <marcela.cunha@liferay.com>

ChrisKian committed:
LPD-26723 Remove reloadUntilVisible method calls, which were there to workaround this bug.

gergelyszalay committed:
LPD-42542 Integration test

Commit:
LPD-22831 Create new service method to update userId for ObjectRelationship, ListTypeDefinition and ListTypeEntry

Co-authored-by: Marcela Cunha <marcela.cunha@liferay.com>

Commit:
LPD-22831 Create UserModelListener to update userId after removing a user for ObjectRelationship, ListTypeDefinition and ListTypeEntry

Co-authored-by: Marcela Cunha <marcela.cunha@liferay.com>

Commit:
LPD-22831 [Integration Test] Create UserModelListenerTest to ensure the userId is being updated after removing a user

Co-authored-by: Marcela Cunha <marcela.cunha@liferay.com>

balazssk committed:
LPD-42985 Add fileEntry level subscription to DLApp

balazssk committed:
LPD-42985 ant build-service-portlet-documentlibrary

balazssk committed:
LPD-42985 Implement subscription wrapper

balazssk committed:
LPD-42985 Add MVCActioncommands for DLFileEntry

balazssk committed:
LPD-42985 semver

balazssk committed:
LPD-42985 Create dropdown option - subscribe/unsubscribe to DLFileEntry

gergelyszalay committed:
LPD-43391 Add Action menu contributor

balazssk committed:
LPD-42985 Create dropdown option - subscribe/unsubscribe to DLFolder

balazssk committed:
LPD-42985 SF rename

balazssk committed:
LPD-42985 Add resource permission

balazssk committed:
LPD-42985 Extend DLSubscriptionUtil with a method that determines if a FileEntry has an explicite or implicite subscription

balazssk committed:
LPD-42985 Add info panel subscription for File Entries

dsanz committed:
LPD-44141 Add method to get pagination deltas from a system data set. Provide default implementations aligned with existing pagination properties

# breaking

## What modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/SystemFDSEntry.java

SystemFDSEntry interface added one method:
   public default int[] getListOfItemsPerPage()
Default implementations is provided, so implementors need to define their own only if a different behavior is required.

It also changes the (default) implementation of public default int getDefaultItemsPerPage() method to leverage property-based delta configurations

## Why

This is needed to accommodate new Data Set definition requirements. It's an internal API not meant to be used outside of DXP.

----

balazssk committed:
LPD-42985 Unsubscribable depends on ancestors, not the current folder

dsanz committed:
LPD-44141 Create FDSBulkActionList registry interface and implementation

balazssk committed:
LPD-42985 Migrate info panel un/subscribe icon to use clay

gergelyszalay committed:
LPD-43391 Integration test for Menu Contributor

dsanz committed:
LPD-44141 Add support for new properties in FDSSortItem and its builder

# breaking

## What modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/model/FDSSortItemBuilder.java

This commit adds 2 new methods to FDSSortItem, that allow to set the active flag and the item label. Consequently, FDSSortItemBuilder class added new interfaces and modified some interfaces to accommodate build-related methods handling the new properties.

Particularly, 2 interfaces have been changed:
  public interface AfterDirectionStep now also extends LabelStep interface
  public interface AfterKeyStep now also extends LabelStep interface

## Why

This is needed to accommodate new Data Set definition requirements. It's an internal API not meant to be used outside of DXP.

----

dsanz committed:
LPD-44141 Create FDSSortList registry interface and implementation

AliciaGarciaGarcia committed:
LPD-43597 only test that a toggle is on or off. not needed

AliciaGarciaGarcia committed:
LPD-43597 add test of multivalued vocabulary on blogs

AliciaGarciaGarcia committed:
LPD-43597 add test to dlAppService to check that if vocabulary is multivalued you can not add 2 categories of this vocabulary

balazssk committed:
LPD-44346 Seperate View File Entry Info Panel from Admin Info Panel

dsanz committed:
LPD-44141 Add support to define props transformer for a system data set

# breaking

## What modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/SystemFDSEntry.java

SystemFDSEntry interface added one method:
   public default String getPropsTransformer()
Default implementations is provided, so implementors just need to implement this method when defining a Data Set that requires a props transformer JS module.

## Why

This is needed to accommodate new Data Set definition requirements. It's an internal API not meant to be used outside of DXP.

----

AliciaGarciaGarcia committed:
LPD-43597 add test of multivalued vocabulary on web content articles

balazssk committed:
LPD-44346 New function to add DocumentFolders

balazssk committed:
LPD-44346 Folder checkbox is at a different part of the DOM

dsanz committed:
LPD-44141 baseline

gislayne-vitorino committed:
LPD-42744 Add language keys

gislayne-vitorino committed:
LPD-42744 Modify notification content to display proper message

kyle-miho committed:
LPD-44535 Check if both the expected and the base files exist

kyle-miho committed:
LPD-44535 Enforce that a unit test be created for each fileType and issueKey replacement in replacements.json

kyle-miho committed:
LPD-44535 Remove redundant unit tests now that parameterizated tests for replacements.json exist

balazssk committed:
LPD-44346 Tests

balazssk committed:
LPD-42985 Root folder is always unsubscribable

AliciaGarciaGarcia committed:
LPD-43597 rename tests to disambiguate which asset we are executing the method on.

cdbm committed:
LPD-44826 make id token optional for refresh token grant

noornajj committed:
LPD-44739 Remove unused api

gislayne-vitorino committed:
LPD-42744 Add tests

balazssk committed:
LPD-44346 SF - use File Entry and Folder in naming

balazssk committed:
LPD-44346 SF - rename

balazssk committed:
LPD-42985 Use monospaced attribute instead of css

brbalazs committed:
LPD-44889 Refactor poshi test CanViewAccountRolesWithPermissions

brbalazs committed:
LPD-44889 Delete poshi test CanViewAccountRolesWithPermissions

rafaprax committed:
LPD-44473 Add feature flag

kyle-miho committed:
LPD-44535 Migrate UpgradeCatchAllCheck.testjava|testjsp checks to LPD_XXXXX.testjava|testjsp files

brookedalton committed:
LPD-44939 Update published web content behavior

brookedalton committed:
LPD-44939 Use apiHelper to create web content

brookedalton committed:
LPD-44939 Use new identifier

jeffwu0724 committed:
LPD-45299 upgrade tomcat to 9.0.98

brookedalton committed:
LPD-44939 Update web content workflow publish process

gislayne-vitorino committed:
LPD-42744 buildLang

rafaprax committed:
LPD-44473 Apply FF

brookedalton committed:
LPD-44939 Assert button is clicked

kyle-miho committed:
LPD-44535 Be more explicit when a unit test is missing

kyle-miho committed:
LPD-44535 SF

noornajj committed:
LPD-44739 Remove related methods and tests

devpedrotavares committed:
LPD-44779 Execute udpate actions when the status of the entry is updated

brookedalton committed:
LPD-44939 Rewrite sandbox test to publish correct publication

ChrisKian committed:
LPD-26723 The SpSessionTerminationSamlPortalFilter must come before the Absolute Redirects Filter, otherwise the ARF may reinitialize the session when it should be invalidated.  But, if we only adjust the order, then we will cause a regression described by LPD-40486 because the SAML filter uses the companyId from CompanyThreadLocal to determine if SAML is enabled or not, and the companyId is set in the ARF.  To fully resolve this issue, the SpSessionTerminationSamlPortalFilter needs to come before the ARF and we need to ensure CompanyThreadLocal is populated with the current companyId based off the request.

noornajj committed:
LPD-44739 Remove old timeline implementation

noornajj committed:
LPD-44739 buildREST

# breaking

## What modules/apps/change-tracking/change-tracking-rest-api/bnd.bnd

Major version increase required

## Why

Unused headless api was removed

----

gergelyszalay committed:
LPD-43391 moving Contributor to util

Mikellorza committed:
LPD-43391 Moving from asset.util to util

amosfong committed:
LRSD-7613 use shorter connection settings to fix connection reset by peer errors

Mikellorza committed:
LPD-43391 To avoid log errors add the site to the themeDisplay and check if exists the friendly url of the layout in the returned url

devpedrotavares committed:
LPD-44779 Add test to assert that onAfterUpdate actions are triggered with workflow status changes

Mikellorza committed:
LPD-45163 Sort parameters

Mikellorza committed:
LPD-45163 Add highlightedDDMStructureId parameter to the base portlet url

rafaprax committed:
LPD-44473 Baseline

cdbm committed:
LPD-44826 add tests for optional id token

brookedalton committed:
LPD-44739 baseline

rafaprax committed:
LPD-1634 Remove useless null checking. According to the current BaseProfile#getMessageContext(HttpServletRequest, HttpServletResponse, String), the messageContext will be never null.

rafaprax committed:
LPD-1634 SF, sort

rafaprax committed:
LPD-1634 SF, LRUMap is still a Map

Mikellorza committed:
LPD-43391 Playwright test for View Usages

rafaprax committed:
LPD-1634 Simplify the _bindSamlSsoRequestContext method extracting the removal logic from the LRU from there. The removal must be done when trying to retrieve the SamlSsoRequestContext from session using a LRU#remove operation

brookedalton committed:
LPD-44739 Add feature flag for tests

brookedalton committed:
LPD-44939 Remove waitForAlert

brookedalton committed:
LPD-44739 Update and remove poshi tests

rafaprax committed:
LPD-1634 Extract repetitive code to utility methods and apply it to usages

rafaprax committed:
LPD-1634 SF, rename. The new cases are testing the decodeAuthnRequest process with concurrent AuthnRequests

rafaprax committed:
LPD-1634 Auto SF

ling-alan-huang committed:
LPD-44842 Rename

ealonso committed:
LPD-42061 Update code to use featureFlagsTest since now it supports system feature flags

michelevigilante committed:
LPD-45186 Fix passed quantity

dsanz committed:
LPD-44141 baseline

lfbesada committed:
LPD-45260 Apply the layout theme instead of the layout set theme

lfbesada committed:
LPD-45260 Set the theme display look and feel depending on the layout theme

rafaprax committed:
LPD-1634 SF, rename

lfbesada committed:
LPD-45260 Add an integration test for this use case

4lejandrito committed:
LPD-45266 Making test more robust to make it fail

The fragment is not being rendered in the page.

4lejandrito committed:
LPD-45266 Avoiding pollution of permission thread local

During the fragment rendering there are some scenarios in which the
request being passed to RESTClient is not authenticated. When that
happens the PermissionThreadLocal is set with the guest user inside the
REST call overriding the previous user.

The issue affects cases where REST Client is run several times within
the same thread.

With these changes we restore the original permission checker after the
REST Call to avoid side effects.

oliv-yu committed:
LPD-45175 portal-web: Update steps for configuring searchbar fragment

ealonso committed:
LPD-45158 Uses another macro

marcelabc committed:
LPD-22831 buildService

Co-authored-by: Marcela Cunha <marcela.cunha@liferay.com>

achaparro committed:
LPD-45295 Simplify by recovering the shortLivedThreadLocals since they longLive ones do not apply for this case

gislayne-vitorino committed:
LPD-42744 Create model listener to add logic to send the notification

achaparro committed:
LPD-45295 We do not need to mantain a list anymore. Ensure all CompanyCentralizedThreadLocal are shortlived

gislayne-vitorino committed:
LPD-42744 Add new notification for when publication changes size classification

ChrisKian committed:
LPD-44473 Build lang

noornajj committed:
LPD-44739 Update tests

noornajj committed:
LPD-44739 Remove failing poshi test since it's being covered by playwright

ling-alan-huang committed:
LPD-44271 Auto SF

ling-alan-huang committed:
LPD-44271 Auto SF

ling-alan-huang committed:
LPD-44271 Auto SF

achaparro committed:
LPD-45295 Sometimes ServiceContextThreadLocal is set to null, take into account that case

magjed4289 committed:
LPD-44402 Do not check details of alert as it can have different outputs that are not all covered with this if-else

Mikellorza committed:
LPD-45163 Add test

carolmariaabb committed:
LPD-22831 [Integration Test] Create StaleUserIdUpgradeProcessTest to ensure the userId is being updated after running the upgrade process

Co-authored-by: Marcela Cunha <marcela.cunha@liferay.com>

cdbm committed:
LPD-44826 change return to void since it is never used

brittneyq committed:
LRCI-5039 Track multiple pipeline names

ChrisKian committed:
LPD-44826 SF Use the expected annotation to handle the exception catching

achaparro committed:
LPD-45295 This was the original behavior, do the safeClosable first to be able to restore the value and remove later to make the initial value set lazy when the get method is called

ling-alan-huang committed:
LPD-43477 SF

ling-alan-huang committed:
LPD-43477 SF

ling-alan-huang committed:
LPD-43477 SF

ling-alan-huang committed:
LPD-43477 SF

gislayne-vitorino committed:
LPD-44274 Specify cursor type to be pointer

gislayne-vitorino committed:
LPD-44274 Add test

ling-alan-huang committed:
LPD-45318 Rename

ChrisKian committed:
LPD-26723 Use WebKeys instead

joseabelenda committed:
LPD-44826 Rename

joseabelenda committed:
LPD-44826 Sort

joseabelenda committed:
LPD-44826 Auto SF

joseabelenda committed:
LPD-44826 As used

joseabelenda committed:
LPD-44826 As used

joseabelenda committed:
LPD-44826 As used

joseabelenda committed:
LPD-44826 Inline

joseabelenda committed:
LPD-44826 Can we sort this?

joseabelenda committed:
LPD-44826 Can we sort this too?

petershin committed:
LPD-44208 Configure bnd bundle

CalumR23 committed:
LRCI-4206 Trim stack trace message for testray

ryanschuhler committed:
LRSD-7768 Fix java version in jar-runner uses

michaelhashimoto committed:
LRCI-5174 Change source compatibility to JDK8 for jenkins-results-parser

brianchandotcom committed:
LPD-1223 SF

michaelhashimoto committed:
LRCI-5173 Change source compatibility to JDK8 for poshi-core & poshi-runner

javalosjr96 committed:
LPD-45337 overload annotation to avoid inheriting the Afterclass method

javalosjr96 committed:
LPD-45337 update Dataguard Scope to CLASS to clear added data

brianchandotcom committed:
LPD-44779 SF

brianchandotcom committed:
LPD-45266 Sort it

brianchandotcom committed:
LPD-42985 Not needed

brianchandotcom committed:
LPD-42985 Regen

brianchandotcom committed:
LPD-42985 SF

brianchandotcom committed:
LPD-42985 SF

brianchandotcom committed:
LRCI-5174 jdk8

brianchandotcom committed:
LPD-43597 SF

michaelhashimoto committed:
LRCI-5173 SF

petershin committed:
LPD-44208 Initialization phase

brianchandotcom committed:
LRCI-5173 prep next

brianchandotcom committed:
LPD-1634 Wordsmith

brianchandotcom committed:
LPD-1634 Regen

brianchandotcom committed:
LPD-1634 Not needed

brianchandotcom committed:
LPD-1634 SF

brianchandotcom committed:
LPD-1634 SF

brianchandotcom committed:
LPD-1634 Can we rename it to this?

brianchandotcom committed:
LPD-1634 Sort

brianchandotcom committed:
LPD-1634 Sort

brianchandotcom committed:
LPD-1634 SF

brianchandotcom committed:
LPD-44473 Sort

brianchandotcom committed:
LPD-44473 Do not hyphenate because it is a noun and not an adjective. See https://languagetips.wordpress.com/category/hyphencompound-words

brianchandotcom committed:
LPD-44473 Regen

david-truong committed:
LPD-42744 consolidate logic

david-truong committed:
LPD-42744 wordsmith

david-truong committed:
LPD-42744 buildLang

dantewang committed:
LPD-45405 Remove the usage of ClearThreadLocalUtil. This was added in the case ServiceBuilder is called to build many services in the same Ant process. Now we call ServiceBuilder with forked JVM and Gradle, so this should no longer be needed.

The usage was added in LPS-15365 3f23a436534d8eb95e637c75d1ecdc0d7580ec6c,
but the build-service was changed to use forked JVM in LPS-27881 930f8c0cbe12f6a782d52721c255f2b539ea839f

dantewang committed:
LPD-45405 Regenerated sem ver

ealonso committed:
LPD-45104 Rename

ealonso committed:
LPD-45104 Auto SF

matyaswollner committed:
LPD-45328 - Refactor LocalFile.AccountPermissions#CanChangePaginationInAccounts into Playwright

matyaswollner committed:
LPD-45328 - Delete LocalFile.AccountPermissions#CanChangePaginationInAccounts

balazssk committed:
LPD-45409 escape user input

smottal committed:
LPD-45411 Remove poshi test

carolmariaabb committed:
LPD-22831 Use get instead of fetch to throw exception if there is no default service account user

dannielraposo committed:
LPD-45327 LPD-44400 Check if start time is strictly after the futureCalendar time

ccorreagg committed:
LPD-45327 LPD-44400 Auto SF

smottal committed:
LPD-44151 Fix test

kevenleone committed:
LPD-44911 Created Connections Table on Customer Dashboard

Co-authored-by: Caleb James Hall <caleb.hall@liferay.com>

kevenleone committed:
LPD-44911 Create Object Definition and Relationship

Co-authored-by: Caleb James Hall <caleb.hall@liferay.com>

sandrodw3 committed:
LPD-42061 Improve locators

kevenleone committed:
LPD-44911 Add missing type

achaparro committed:
LPD-45337 Not needed

achaparro committed:
LPD-45337 Scope CLASS is the default bahavior

achaparro committed:
LPD-45337 SF

eduardo-diniz committed:
LPD-44835 Create authorize route

eduardo-diniz committed:
LPD-44835 Create Authorize Page Definition

eduardo-diniz committed:
LPD-44835 Add new translate

pyoo47 committed:
LRCI-5039 Use getNounForm()

eduardo-diniz committed:
LPD-44835 Create DxpOAuth2Client

eduardo-diniz committed:
LPD-44835 Update some methods

eduardo-diniz committed:
LPD-44835 Create DXP connection flow

shuyangzhou committed:
LPD-45467 Forward compatibility, remove @Override for removed servlet apis.

shuyangzhou committed:
LPD-45467 Fix unit test

shuyangzhou committed:
LPD-45469 Forward compatibility, avoid directly implementing ServletContext

petershin committed:
LPD-44208 Fix extractGradle tasks

ryanschuhler committed:
LRSD-6922 Remove flakey tests

JhossephG committed:
LPD-44524 Update method test name

brianchandotcom committed:
LPD-44303 auto SF

brianchandotcom committed:
LRSD-7768 apply

dantewang committed:
LPD-X Use maven-plugin-plugin 3.6.4

kevenleone committed:
LPD-44835 Add Translations

kevenleone committed:
LPD-44835 Update Account Selection Props

kevenleone committed:
LPD-44835 Split logic from the router and the outlet

kevenleone committed:
LPD-44835 Rename OAuth2Authorize pages and use outlet context instead of props

kevenleone committed:
LPD-44835 No longer needed

kevenleone committed:
LPD-44835 SF

brianchandotcom committed:
LPD-X prep next

brianchandotcom committed:
LPD-45405 prep next

brianchandotcom committed:
LPD-45405 prep next

brianchandotcom committed:
LRCI-4206 SF

brianchandotcom committed:
LRCI-4206 prep next

brianchandotcom committed:
LPD-44524 auto SF

brianchandotcom committed:
LPD-42744 Buy space

brianchandotcom committed:
LRCI-5039 SF

brianchandotcom committed:
LRCI-5039 prep next

brianchandotcom committed:
LPD-45295 Rename

brianchandotcom committed:
LPD-45295 auto SF

brianchandotcom committed:
LRSD-7613 And make it Java 8 compat

brianchandotcom committed:
LRSD-7613 prep next

brianchandotcom committed:
Revert "LRSD-7768 Fix java version in jar-runner uses"

This reverts commit 1b969bed8e7195905d327e1a584bc7d28d9d6012.

brianchandotcom committed:
Revert "LRSD-7768 apply"

This reverts commit 1fb44ca925a57c2d562732f4572ae20e797bca7c.

brianchandotcom committed:
LPD-44826 As used

brianchandotcom committed:
LPD-44826 Move to a method

brianchandotcom committed:
LPD-44826 Inline

brianchandotcom committed:
LPD-44826 SF

liferay-translation committed:
LPS-77699 Update Translations

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
ci:test:playwright

liferay-continuous-integration commented:
<html><h3>:x: ci:test:playwright - 0 out of 1 jobs passed in 8 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/61b58512ab7f586f7ca6824d9a013acf90aafc3f">61b58512ab7f586f7ca6824d9a013acf90aafc3f</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/83a242f48ab4b2086965005a590a95e42e9c0a13">83a242f48ab4b2086965005a590a95e42e9c0a13</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/103654631">EE Development Acceptance (master) - 1117 - 2025-01-04[12:36:18]</a></p><details><summary><strong>ci:test:playwright - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/12094/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12094/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/12094/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre>Started by remote host 172.16.168.126
Running as SYSTEM
[EnvInject] - Loading node environment variables.
Building remotely on test-1-28-77 (slave-1) in workspace /opt/dev/projects/github/liferay-jenkins-ee
[liferay-jenkins-ee] \$ /bin/bash -e -x /tmp/jenkins7531236027180735235.sh
+ echo 'Initialize Jenkins slave node.'
Initialize Jenkins slave node.
+ cd /opt/dev/projects/github/liferay-jenkins-ee
+ curl -o ./_common.sh http://mirrors-no-cache.lax.liferay.com/github.com/liferay/liferay-jenkins-ee/_common.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 19380  100 19380    0     0  2235k      0 --:--:-- --:--:-- --:--:-- 2365k
+ source ./_common.sh
++ MAX_THREADS=1
++ [[ test-1-28-77.lax.liferay.com =~ cloud-.* ]]
++ [[ test-1-28-77 slave-1 =~ .*win.* ]]
++ ROOT_DIR=
++ GIT_REPOSITORIES_ROOT_DIR=/opt/dev/projects/github
++ JENKINS_REPO_PATH=/opt/dev/projects/github/liferay-jenkins-ee
+++ cat /opt/dev/projects/github/liferay-jenkins-ee/git-directories.json
++ GIT_DIRECTORIES_JSON_CONTENT='[
  \{
    "branch": "master",
    "name": "alloy-ui",
    "repository": "alloy-ui",
    "type": "mounted"
  \},
  \{
    "branch": "7.0.x",
    "name": "com-liferay-osb-asah-private",
    "repository": "com-liferay-osb-asah-private",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-binaries-cache-2017",
    "repository": "liferay-binaries-cache-2017",
    "type": "shared-mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-binaries-cache-2020",
    "repository": "liferay-binaries-cache-2020",
    "type": "shared-mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-blade-samples",
    "repository": "liferay-blade-samples",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-docker",
    "repository": "liferay-docker",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-alloy",
    "repository": "liferay-faces-alloy",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-bridge-impl",
    "repository": "liferay-faces-bridge-impl",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-portal",
    "repository": "liferay-faces-portal",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-showcase",
    "repository": "liferay-faces-showcase",
    "type": "mounted"
  \},
  \{
    "branch": "master",</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

sandrodw3 commented:
ci:test:playwright

liferay-continuous-integration commented:
<html><h3>:x: ci:test:playwright - 0 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/61b58512ab7f586f7ca6824d9a013acf90aafc3f">61b58512ab7f586f7ca6824d9a013acf90aafc3f</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/83a242f48ab4b2086965005a590a95e42e9c0a13">83a242f48ab4b2086965005a590a95e42e9c0a13</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/103654631">EE Development Acceptance (master) - 1117 - 2025-01-04[12:36:18]</a></p><details><summary><strong>ci:test:playwright - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/13539/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13539/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/13539/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre>Started by remote host 172.16.168.126
Running as SYSTEM
[EnvInject] - Loading node environment variables.
Building remotely on test-1-27-31 (slave-1) in workspace /opt/dev/projects/github/liferay-jenkins-ee
[liferay-jenkins-ee] \$ /bin/bash -e -x /tmp/jenkins16860785579134128415.sh
+ echo 'Initialize Jenkins slave node.'
Initialize Jenkins slave node.
+ cd /opt/dev/projects/github/liferay-jenkins-ee
+ curl -o ./_common.sh http://mirrors-no-cache.lax.liferay.com/github.com/liferay/liferay-jenkins-ee/_common.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 19380  100 19380    0     0  1828k      0 --:--:-- --:--:-- --:--:-- 1892k
+ source ./_common.sh
++ MAX_THREADS=1
++ [[ test-1-27-31.lax.liferay.com =~ cloud-.* ]]
++ [[ test-1-27-31 slave-1 =~ .*win.* ]]
++ ROOT_DIR=
++ GIT_REPOSITORIES_ROOT_DIR=/opt/dev/projects/github
++ JENKINS_REPO_PATH=/opt/dev/projects/github/liferay-jenkins-ee
+++ cat /opt/dev/projects/github/liferay-jenkins-ee/git-directories.json
++ GIT_DIRECTORIES_JSON_CONTENT='[
  \{
    "branch": "master",
    "name": "alloy-ui",
    "repository": "alloy-ui",
    "type": "mounted"
  \},
  \{
    "branch": "7.0.x",
    "name": "com-liferay-osb-asah-private",
    "repository": "com-liferay-osb-asah-private",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-binaries-cache-2017",
    "repository": "liferay-binaries-cache-2017",
    "type": "shared-mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-binaries-cache-2020",
    "repository": "liferay-binaries-cache-2020",
    "type": "shared-mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-blade-samples",
    "repository": "liferay-blade-samples",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-docker",
    "repository": "liferay-docker",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-alloy",
    "repository": "liferay-faces-alloy",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-bridge-impl",
    "repository": "liferay-faces-bridge-impl",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-portal",
    "repository": "liferay-faces-portal",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-showcase",
    "repository": "liferay-faces-showcase",
    "type": "mounted"
  \},
  \{
    "branch": "master",</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>
